### PR TITLE
[Power] HP40A-126: boot up from power loss to last mode

### PR DESCRIPTION
**Pull request by arun-madhavan-013**
Reason for change: Added logic to store lastpowermode in uboot environment variable so that device can boot to last power state when powerloss recovery happens when in that state.
Test Procedure: Build complete image and test Power plugin.
Risks: Medium.

Signed-off-by: Arun P Madhavan <arun_madhavan@comcast.com>

### CODE REVIEW

File: cmake/FindMFRBootenvlibs.cmake [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# - Try to find MFRBOOTENVLIBS
```
Comment (mhughesacn) [inline cmake/FindMFRBootenvlibs.cmake:1]: Hi Arun, please place the RDK header at the top of the file, not starting at line 6.
Thanks, Martin, RDK CMF Compliance team

**Comment by mhughesacn** [comment]
Hi Arun, Please place the RDK header in the cmake file at the top of the file in the .cmake file, not at line 6.  Thanks,
Martin Hughes, RDK CMF Compliance Team

**Comment by arun-madhavan-013** [comment]
> Hi Arun, Please place the RDK header in the cmake file at the top of the file in the .cmake file, not at line 6. Thanks,
> Martin Hughes, RDK CMF Compliance Team

Updated as suggested. Thanks.

**Comment by arun-madhavan-013** [comment]
Abandoning this PR to add a generic implementation.

**Pull request closed without merging**